.github/workflows/build.yml

```diff
@@ -74,7 +75,7 @@ jobs:
     - name: Install Python
       uses: actions/setup-python@v2
       with:
-        python-version: '3.9'
+        python-version: '3.8'
```

Comment:
I put here 3.8 for the isolated test, as that is the current version in EBRAINS LAB.